tm2/pkg/bft/config/config.go

```diff
@@ -24,6 +25,7 @@ type Config struct {
 	P2P       *p2p.P2PConfig       `toml:"p2p"`
 	Mempool   *mem.MempoolConfig   `toml:"mempool"`
 	Consensus *cns.ConsensusConfig `toml:"consensus"`
+	Indexer   *config.Config       `toml:"indexer"`
```

Comment:
For consistency with other imports, maybe alias this one to `txindex` ?

```go
txindex "github.com/gnolang/gno/tm2/pkg/bft/state/txindex/config"
```

Reply:
Good catch 💯 

Standardized the import alias:
[2c5aaa6](https://github.com/gnolang/gno/pull/546/commits/2c5aaa64c80e5edd8259cb34d9360529f49528d2)